python/taichi/lang/util.py

```diff
@@ -11,7 +11,7 @@
 
 _has_pytorch = False
 
-_env_torch = os.environ.get('TI_ENABLE_TORCH', '0')
+_env_torch = os.environ.get('TI_ENABLE_TORCH', '1')
 if not _env_torch or int(_env_torch):
```

Comment:
do we still need this env?

Reply:
@ailzhang, yes, we have like less than 20 tests for torch io etc. This env provides possibilities to enable or disable these tests. I think it's better to leave it here.